tello Test/tello_web_server.py

```
@@ -651,15 +740,650 @@ def _log_operation(self, operation: str, details: Dict[str, Any]):
+                            break
+                    except Exception as reinit_e:
+                        logger.error(f"ビデオキャプチャ再初期化エラー: {reinit_e}")
+                        break
+                else:
+                    logger.error(f"フレームキャプチャエラー: {e}")
+                
+                consecutive_failures += 1
+                if consecutive_failures >= max_failures:
+                    logger.error("フレームキャプチャエラーが多すぎるため、スレッドを終了します")
+                    break
+                time.sleep(0.1)
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)